### PR TITLE
feat: tabs component

### PR DESCRIPTION
**Pull request by jessywork**
# Description

This update addresses the need for a customizable and reusable set of tabs component in the project.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/components/tabs/src/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{ onClick: (value: string) => void }) => {
+  const handleClick = () => {
+    if (!disabled) {
+      onClick(value);
+    }
+  };
+
+  return (
+    <TabStyled 
+        className={selected ? 'active' : ''}
```
Comment (joaoprcosta) [inline packages/components/tabs/src/Tabs.tsx:21]: Check the example in this tutorial.
Managing conditional styles is more easy than this.

**Review by JuniorDiasOliveira — APPROVED** [review]

File: packages/components/tabs/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export { default as tabs } from './Tabs';
```
Comment (joaoprcosta) [inline packages/components/tabs/src/index.ts:1]: Tabs must be Uppercase.
You forgot to export TabsGroup.

**Comment by joaoprcosta** [comment]
The structure of the component is not aligned with the expected and will not generate the first version of the package. Check the button component to see an example.

**Review by joaoprcosta — APPROVED** [review]

File: packages/components/tabs/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export { default as Tabs } from './Tabs';
```
Comment (joaoprcosta) [inline packages/components/tabs/src/index.ts:1]:
```suggestion
export { Tabs } from './Tabs';
export { TabsGroup } from './TabsGroup';
export * from './tabs.types';
```

File: packages/components/tabs/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+```js
+import { Tabs, TabsGroup } from '@douro-ui/tabs';
+import type { TabsProps, TabsGroupProps } from '@douro-ui/tabs';
+```
+
+or
+
+```js
+import Tabs, { TabsProps } from '@douro-ui/tabs';
```
Comment (joaoprcosta) [inline packages/components/tabs/README.md:34]:
```suggestion
import { Tabs, TabsProps } from '@douro-ui/tabs';
```

**Pull request merged**